### PR TITLE
Iow 820 - allow for selecting parameters that are only 'field visit' groundwater levels

### PR DESCRIPTION
**Pull request by mbucknell**
Before making a pull request
----------------------------

- [X] Make sure all tests run
- [X] Update the changelog appropriately

Title
-----------
 Allow for selecting parameters that are only 'field visit' groundwater levels

Description
-----------
I am now adding the groundwater variables to the list of variables in ivTimeSeriesData in the Redux store. This was the most
expedient way to allow display of these parameters in the parameter selection table. If we do the refactor of the way we store IV data we should think about separating out the data from the metadata for a particular kind of data.

I also added a new configuration parameter , SITE_SERVICE_CATALOG_ROOT so that I could use our QA service for the service catalog while continuing to use production services for the IV data retrieval. The QA service has the new groundwater level data. For production, this should be set to the production service and should be the same as SERVICE_ROOT.

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: assets/src/scripts/monitoring-location/components/hydrograph/selectors/parameter-data.js [hunk 1]
```diff
@@ -16,24 +18,46 @@ export const getAvailableParameterCodes = createSelector(
...
+                if (hasWaterAlert) {
+                    if (measuredParameterCode === parameterCode) {
+                        waterAlertDisplayText = 'Subscribe';
+                    } else {
+                        waterAlertDisplayText = 'Alerts in C';
+                    }
+                } else {
+                    waterAlertDisplayText = 'N/A';
+                }
+
```
[the remark this answers is not in the document]
Comment (abriggs-usgs) [reply assets/src/scripts/monitoring-location/components/hydrograph/selectors/parameter-data.js:42]: I really like the move to consolidate more of the meta data type information into one object. I think it not only reduces the hunting around for the information, but makes the code more concise and easier to understand. I found myself tinkering around with this section of code when I was looking at plotting the second parameter code.

File: assets/src/scripts/monitoring-location/store/instantaneous-value-time-series-data.js [hunk 2]
```diff
@@ -371,6 +365,13 @@ export const ivTimeSeriesDataReducer = function(ivTimeSeriesData={}, action) {
             return newSeries;
         }
 
+        case 'ADD_VARIABLE_TO_IV_VARIABLES': {
```
Comment (abriggs-usgs) [inline assets/src/scripts/monitoring-location/store/instantaneous-value-time-series-data.js:368]: I like this way of combining the new variable with the rest.

**Review by abriggs-usgs — APPROVED** [review]
Looks good to me. I really like having more of the information related to 'variables' in one object, and how the actual combing of the state 'variables' is done in an action.

File: assets/src/scripts/monitoring-location/components/hydrograph/index.js [hunk 3]
```diff
@@ -116,25 +112,36 @@ export const attachToNode = function(store,
...
+                const thisVariable = Object.values(getVariables(state)).find(isThisParamCode);
+                if (thisVariable) {
+                    store.dispatch(ivTimeSeriesStateActions.setCurrentIVVariable(thisVariable.oid));
+                }
+            }
+            if (!getCurrentVariableID(state)) {
+                //Sort variables and use the first one as the current variable
+                const sortedVars = sortedParameters(getVariables(state));
+                if (sortedVars.length) {
+                    console.log('Setting current variable ' + sortedVars[0].oid);
```
[the remark this answers is not in the document]
Comment (mbucknell) [reply assets/src/scripts/monitoring-location/components/hydrograph/index.js]: Nope. Thanks I'll remove it

**Pull request merged**